### PR TITLE
Cbrelease 4.8.11.1

### PR DESCRIPTION
**Pull request by NandiniAV**
New translations fixes & labels & Merged 4.8.11 code

### CODE REVIEW

File: src/app/routes/public/public-request/public-request.component.ts [hunk 1]
```diff
@@ -216,14 +216,14 @@ export class PublicRequestComponent implements OnInit {
     if (mob && mob.value && Math.floor(mob.value) && mob.valid) {
       this.signupSvc.sendOtp(mob.value, 'phone').subscribe(() => {
         this.otpSend = true
-        alert('An OTP has been sent to your mobile number (valid for 15 minutes)')
+        alert(this.translateLabels('anOtpHasBeenSentToMobile', 'publicsignup'))
```
Comment (vishnubansaltarento) [inline src/app/routes/public/public-request/public-request.component.ts:212]: alert should be here?

File: src/app/routes/public/public-request/public-request.component.ts [hunk 2]
```diff
@@ -233,16 +233,15 @@ export class PublicRequestComponent implements OnInit {
         if ((_.get(res, 'result.response')).toUpperCase() === 'SUCCESS') {
           this.otpSend = true
           this.disableVerifyBtn = false
-
-          alert('An OTP has been sent to your mobile number (valid for 15 minutes)')
+          alert(this.translateLabels('anOtpHasBeenSentToMobile', 'publicsignup'))
```
Comment (vishnubansaltarento) [inline src/app/routes/public/public-request/public-request.component.ts:229]: alert should be here?

File: src/app/routes/public/public-request/public-request.component.ts [hunk 3]
```diff
@@ -343,15 +342,15 @@ export class PublicRequestComponent implements OnInit {
         if ((_.get(res, 'result.response')).toUpperCase() === 'SUCCESS') {
           this.otpEmailSend = true
           this.disableEmailVerifyBtn = false
-          alert('An OTP has been sent to your email address (valid for 15 minutes)')
+          alert(this.translateLabels('anOtpHasBeenSentToEmail', 'publicsignup'))
```
Comment (vishnubansaltarento) [inline src/app/routes/public/public-request/public-request.component.ts:338]: alert should be here?

File: src/app/routes/public/public-signup/public-signup.component.ts [hunk 4]
```diff
@@ -411,14 +411,14 @@ export class PublicSignupComponent implements OnInit, OnDestroy {
     if (mob && mob.value && Math.floor(mob.value) && mob.valid) {
       this.signupSvc.sendOtp(mob.value, 'phone').subscribe(() => {
         this.otpSend = true
-        alert('An OTP has been sent to your mobile number (valid for 15 minutes)')
+        alert(this.translateLabels('anOtpHasBeenSentToMobile', 'publicsignup'))
```
Comment (vishnubansaltarento) [inline src/app/routes/public/public-signup/public-signup.component.ts:406]: alert should be here?

File: src/app/routes/public/public-signup/public-signup.component.ts [hunk 5]
```diff
@@ -429,15 +429,15 @@ export class PublicSignupComponent implements OnInit, OnDestroy {
         if ((_.get(res, 'result.response')).toUpperCase() === 'SUCCESS') {
           this.otpSend = true
           this.disableVerifyBtn = false
-          alert('An OTP has been sent to your mobile number (valid for 15 minutes)')
+          alert(this.translateLabels('anOtpHasBeenSentToMobile', 'publicsignup'))
```
Comment (vishnubansaltarento) [inline src/app/routes/public/public-signup/public-signup.component.ts:424]: alert should be here?

File: src/app/routes/public/public-request/public-request.component.ts (hunk 2 above)
Comment (NandiniAV) [reply src/app/routes/public/public-request/public-request.component.ts:229]: yes

File: src/app/routes/public/public-request/public-request.component.ts (hunk 3 above)
Comment (NandiniAV) [reply src/app/routes/public/public-request/public-request.component.ts:338]: yes

File: src/app/routes/public/public-signup/public-signup.component.ts (hunk 4 above)
Comment (NandiniAV) [reply src/app/routes/public/public-signup/public-signup.component.ts:406]: yes

File: src/app/routes/public/public-signup/public-signup.component.ts (hunk 5 above)
Comment (NandiniAV) [reply src/app/routes/public/public-signup/public-signup.component.ts:424]: yes

**Pull request merged**